src/ContaoManager/Plugin.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2016 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\FaqBundle\ContaoManager;
+
+use Contao\ManagerBundle\ContaoManager\Bundle\BundleConfig;
+use Contao\ManagerBundle\ContaoManager\Bundle\BundlePluginInterface;
+use Contao\ManagerBundle\ContaoManager\Bundle\ParserInterface;
+
+/**
+ * Plugin for the Contao Manager.
+ *
+ * @author Andreas Schempp <https://github.com/aschempp>
+ */
+class Plugin implements BundlePluginInterface
+{
+    /**
+     * @inheritdoc
+     */
+    public function getBundles(ParserInterface $parser)
+    {
+        return [
```

Comment:
I thought about this, but I don't think it's a good idea. It will allow ambiguous return values, which are harder to type hint and analyze. Also, we should do best practice in the core, and if people read this code they will understand the return value is an array and can be multiple configs.